internal/cloudapi/openapi.yml

```diff
@@ -192,6 +192,7 @@ components:
       required:
         - access_key_id
         - secret_access_key
+        - share_with_accounts
```

Comment:
Hm one thing to consider is if this should be a required property, wdyt @teg ?

Reply:
I think it should be optional as it is now. From image-builder it would always be used, but we could have other services consume this API that does not use the same model (some CI service, for instance?).

Reply:
Ah, in that case i'll repush tomorrow without it being a required property. Right now it will return bad request if you omit it.

Reply:
Thanks!